logic/vote.js

```diff
@@ -227,14 +227,10 @@ Vote.prototype.verifyVote = function(vote, cb) {
 		return setImmediate(cb, 'Invalid vote type');
 	}
 
-	if (!/[-+]{1}[0-9a-z]{64}/.test(vote)) {
+	if (!/^[-+]{1}[0-9a-f]{64}$/.test(vote)) {
```

Comment:
`{1}` is redundant here

Reply:
I think it makes the regex more descriptive. What do you think?